remo/featuredrep/views.py

```diff
@@ -10,11 +11,25 @@
 from models import FeaturedRep
 
 
-@cache_control(private=True, max_age=60*10)
+@cache_control(private=True, max_age=60 * 10)
 def list_featured(request):
     """List all Featured Reps."""
+    featured_list = FeaturedRep.objects.all().order_by('-created_on')
```

Comment:
I guess filtering both by either `created_on` or `updated_on` has it's advantages and disadvantages. Either way, maybe it's better to add this in the `Meta` class and not here so that we can have a single place where the order of the entries in the queryset is determined.